app/views/peerage/peers/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if signed_in? && current_user.sca_name == @peer.sca_name %>
-  <%= link_to('Edit Profile', '/users/edit') %>
+  <%= link_to('Edit Profile', users_edit_path(peerage: peerage)) %>
```

Comment:
It seems like this is a slight change in behavior, so it should be accompanied by a test verifying the expected behavior here - a view test would probably be sufficient

Reply:
Did this, but with a request spec because stubbing everything out of the view was too hard.